src/backend/storeManagers/legendary/commands/egl_sync.ts

```diff
@@ -0,0 +1,16 @@
+import { Path } from './base'
+
+interface EglSyncCommand {
+  subcommand: 'egl-sync'
+  '--enable-sync'?: true
+  '--egl-wine-prefix'?: Path
+  '--disable-sync'?: true
+  '--egl-manifest-path'?: Path
+  '--one-shot'?: true
+  '--import-only'?: true
+  '--export-only'?: true
+  '--migrate'?: true
+  '--unlink'?: true
+}
```

Comment:
This is a *super* minor thing, but for the other commands I've added parameters in the order they're displayed in Legendary's README/the output of `-h` (in this case, first `--egl-manifest-path`, then `--egl-wine-prefix`, and so on)